### PR TITLE
Solaris Build

### PR DESCRIPTION
**Pull request by davidbernard**
Minor changes to rocksdb to allow it to build on solaris with gcc49.

### CODE REVIEW

**Comment by davidbernard** [comment]
Withdraw as build problem with windows

**Comment by davidbernard** [comment]
Clean build on Mac OS X and Linux. Windows issues should be fixed.

File: .travis.yml [hunk 1]
```diff
@@ -38,6 +38,4 @@ script:
 
 notifications:
     email:
-      - leveldb@fb.com
-    webhooks:
-      - https://buildtimetrend.herokuapp.com/travis
+      - davidbernard@optushome.com.au
```
Comment (siying) [inline .travis.yml]: I don't get it.

Comment (davidbernard) [inline .travis.yml]: Reverted

File: util/options_builder.cc [hunk 2]
```diff
@@ -7,6 +7,8 @@
 #include <algorithm>
 #include "rocksdb/options.h"
 
+using std::log;
```
Comment (siying) [inline util/options_builder.cc]: We follow Google C++ style, which forbids using namespace: https://google.github.io/styleguide/cppguide.html#Namespaces "Named Namespaces" section.

Is there a way you can spell out where the namespaces are used?

Same comment for utilities/geodb/geodb_impl.cc.

**Comment by siying** [comment]
@davidbernard appreciate your awesome contribution! I'll merge as soon as the review comments are addressed.

File: util/options_builder.cc (hunk 2 above)
Comment (davidbernard) [inline util/options_builder.cc]: Removed "using std::log" and "using std::floor" and added "std::" to "log" and "floor" calls.

**Comment by siying** [comment]
Thank you for your contribution!

**Pull request merged**